Header header
float32 Temperature #In degrees Celsius
float32 OutputCurrent #In Amperes
float32 BusVoltage #In volts
float32 ClosedLoopError #In degrees